recipes/hug/LICENSE

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
```

Comment:
Yes, keep it since the file has the same name as in the `MANIFEST.in`:

https://github.com/timothycrosley/hug/blob/develop/LICENSE
https://github.com/timothycrosley/hug/blob/develop/MANIFEST.in